solidity/random-beacon/tasks/staking.ts

```diff
@@ -0,0 +1,110 @@
+import { task, types } from "hardhat/config"
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+task(
+  "staking",
+  "Stakes T tokens, increases authorization and registers the operator "
+)
+  .addOptionalParam("owner", "Staking owner address", undefined, types.string)
+  .addOptionalParam("provider", "Staking provider", undefined, types.string)
```

Comment:
Hardhat doesn't allow for camel cases. It requires optional args to be all lowercase. Since this is a staking file and all the things here are around staking, I assumed it would be fine naming params just "owner", "provider" etc. 

Same comment for the similar comments below.

Reply:
Hardhat doesn't allow for camel cases. It requires optional args to be all lowercase. Since this is a staking file and all the things here are around staking, I assumed it would be fine naming params just "owner", "provider" etc. 

Same comment for the similar comments below.